documentation/source/_static/custom.css

```diff
@@ -2,11 +2,51 @@
 
 div.sidebarlogo {
     text-align: center;
-    padding: 10px 15px 20px 10px;
+    padding: 20px;
 }
 
 pre {
     border: 1px solid #F04040;
     border-left: none;
     border-right: none;
 }
+
+div.sphinxsidebarwrapper {
```

Comment:
There are missing page things next to the links

Reply:
I'm not quite sure what you mean. We can also talk about this in person.

Reply:
Were you referring to the missing pictures? I just realized that the icons did not get committed because of the .gitignore file. It should be fixed now.